### PR TITLE
feat(clcore): Adding Missing Vehicles

### PR DESCRIPTION
**Pull request by Demo4889**
### Goal of this PR
Updating vehicle list

### How is this PR achieving the goal
Adding missing vehicles

### This PR applies to the following area(s)
FiveM

### CODE REVIEW

**Comment by Demo4889** [comment]
@PsychoShock

I wasn't able to squash my commits due to merges being in there. So I just deleted my entire fork, forked it again, and pasted my changes into the file lol...

It's running all the checks now.

**Comment by operationairstrike** [comment]
Ty bro

**Review by martonp96 — APPROVED** [review]

**Comment by prikolium-cfx** [comment]
How does it happen that your enum has two Riots with different hashes?

**Comment by Demo4889** [comment]
> How does it happen that your enum has two Riots with different hashes?

I can check into it later. Just got to work.

File: code/client/clrcore/External/VehicleHash.cs [hunk 1]
```diff
@@ -287,78 +477,134 @@ public enum VehicleHash : uint
...
 		RentalBus = 3196165219u,
+		Retinue = 1841130506u,
+		Retinue2 = 2031587082u,
+		Revolter = 3884762073u,
 		Rhapsody = 841808271u,
+		Rhinehart = 2439462158u,
 		Rhino = 782665360u,
+		Riata = 2762269779u,
 		Riot = 3089277354u,
+		Riot = 22601952180u,
```
Comment (PsychoShock) [inline code/client/clrcore/External/VehicleHash.cs:590]:
```suggestion
		Riot2 = 2601952180u,
```

Small mistake here :)

**Review by PsychoShock — COMMENTED** [review]
It should fix the "2 riot" issue.

**Comment by prikolium-cfx** [comment]
Can I ask you to compile it on your own machine before pushing as PR? In C, C++ and C# variable and enum value name can't start from number.

**Pull request closed without merging**